facebook/delphiFacebook/R/contingency_indicators.R

```diff
@@ -68,14 +68,64 @@ get_aggs <- function() {
 
   indicators <- tribble(
     ~name, ~metric, ~compute_fn, ~post_fn,
+    # behavior
+    ## Mask wearing and distancing
     "pct_wearing_mask_5d", "c_mask_often", compute_binary, jeffreys_binary,
     "pct_wearing_mask_7d", "c_mask_often_7d", compute_binary, jeffreys_binary,
+    "pct_others_masked_public", "c_others_masked_public", compute_binary, jeffreys_binary,
+    "pct_others_distanced_public", "c_others_distanced_public", compute_binary, jeffreys_binary,
+    
     "pct_cli", "hh_p_cli", compute_household_binary, jeffreys_count,
     "pct_ili", "hh_p_ili", compute_household_binary, jeffreys_count,
     "pct_hh_cmnty_cli", "hh_community_yes", compute_binary, jeffreys_binary,
     "pct_direct_contact", "c_direct_contact", compute_binary, jeffreys_binary,
-    "pct_anosmia", "indiv_anosmia", compute_binary, jeffreys_binary,
     
+    # symptoms
+    "pct_symp_fever", "symp_fever", compute_binary, jeffreys_binary,
+    "pct_symp_cough", "symp_cough", compute_binary, jeffreys_binary,
+    "pct_symp_shortness_breath", "symp_shortness_breath", compute_binary, jeffreys_binary,
+    "pct_symp_diff_breathing", "symp_diff_breathing", compute_binary, jeffreys_binary,
+    "pct_symp_fatigue", "symp_fatigue", compute_binary, jeffreys_binary,
+    "pct_symp_nasal_congestion", "symp_nasal_congestion", compute_binary, jeffreys_binary,
+    "pct_symp_runny_nose", "symp_runny_nose", compute_binary, jeffreys_binary,
+    "pct_symp_aches", "symp_aches", compute_binary, jeffreys_binary,
+    "pct_symp_sore_throat", "symp_sore_throat", compute_binary, jeffreys_binary,
+    "pct_symp_chest_pain", "symp_chest_pain", compute_binary, jeffreys_binary,
+    "pct_symp_nausea", "symp_nausea", compute_binary, jeffreys_binary,
+    "pct_symp_diarrhea", "symp_diarrhea", compute_binary, jeffreys_binary,
+    "pct_anosmia", "symp_loss_smell_taste", compute_binary, jeffreys_binary,
```

Comment:
Does this one not match the naming of the others simply because it predates the better naming scheme?

Reply:
That's correct. I can mirror `anosmia` as `symp_loss_smell_taste` (either by running the aggregation procedure twice, or by copying the results over explicitly at some point), or rename it and discontinue `anosmia`. Early on in the contingency tables we renamed some other signals so in the docs those old names are listed as "formerly known as".

Reply:
I think the right decision here would be to check with UMD on their naming, to ensure consistency; I don't think it matters too much what the exact name is

Reply:
There's a mix of names on the UMD side. They offer this signal as `anosmia` *and* `symp_loss_smell_taste`, but a related indicator (based on B2c, which symptoms are unusual) only as `anosmia_unusual`. I'll leave ours as `anosmia` for better consistency between all the related signals.